### PR TITLE
Include BEIS Waste data

### PR DESCRIPTION
**Pull request by greenercode**
**Is this a new addition or an update to an existing factor?**

- [x] New

Waste

**What?**

Include waste disposal emission factors from construction (BEIS, 2021). 

**Why?**

The inclusion of the BEIS waste data is essential for the construction sector.

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -1277,3 +1277,142 @@ Energy,Fuel,fuel_type_coking_coal-fuel_use_na,Coking coal,kg-CO2e/t,3165.24,,BEI
...
+Waste,Construction Waste,waste_type_commercial_and_industrial-waste_use_disposal,Commercial and industrial waste,kg-CO2e/t,467,,BEIS,2021,GB,2021/11/18,Emission intensity for commercial and industrial waste (landfill). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,landfill
+Waste,Construction Waste,waste_type_weee_fridges_and_freezers-waste_use_disposal,WEEE - fridges and freezers,kg-CO2e/t,21.3,,BEIS,2021,GB,2021/11/18,Emission intensity for WEEE - fridges and freezers (open-loop). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,open-loop
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1342]: Please provide full description of "WEEE" in description field for these EFs (1342-1352)

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -1277,3 +1277,142 @@ Energy,Fuel,fuel_type_coking_coal-fuel_use_na,Coking coal,kg-CO2e/t,3165.24,,BEI
...
+Waste,Construction Waste,waste_type_average_plastic_rigid-waste_use_disposal,Average plastics rigid,kg-CO2e/t,8.9,,BEIS,2021,GB,2021/11/18,Emission intensity for average plastics rigid (landfill). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,landfill
+Waste,Construction Waste,waste_type_hdpe_including_forming-waste_use_disposal,HDPE (incl. forming),kg-CO2e/t,21.3,,BEIS,2021,GB,2021/11/18,Emission intensity for HDPE including forming (open-loop). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,open-loop
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1383]: Please provide full terms for "HDPE", "LDPE", "LLDPE", "PET", "PP" and "PVC" for lines 1383-1406

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -1277,3 +1277,142 @@ Energy,Fuel,fuel_type_coking_coal-fuel_use_na,Coking coal,kg-CO2e/t,3165.24,,BEI
...
+Waste,Construction Waste,waste_type_wood-waste_use_disposal,Wood,kg-CO2e/t,9,,BEIS,2021,GB,2021/11/18,Emission intensity for wood waste (composting). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,composting
+Waste,Construction Waste,waste_type_wood-waste_use_disposal,Wood,kg-CO2e/t,828,,BEIS,2021,GB,2021/11/18,Emission intensity for wood waste (landfill). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,landfill
+Waste,Construction Waste,waste_type_books-waste_use_disposal,Books,kg-CO2e/t,21.3,,BEIS,2021,GB,2021/11/18,Emission intensity for books waste (closed-loop). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,closed-loop
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1309]: From lines 1309-1318, BEIS category is "Other" - suggest this be "General waste" to be clearer

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -1277,3 +1277,142 @@ Energy,Fuel,fuel_type_coking_coal-fuel_use_na,Coking coal,kg-CO2e/t,3165.24,,BEI
...
+Waste,Construction Waste,waste_type_clothing-waste_use_disposal,Clothing,kg-CO2e/t,444.9,,BEIS,2021,GB,2021/11/18,Emission intensity for clothing waste (landfill). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,landfill
+Waste,Construction Waste,waste_type_household_residual_waste-waste_use_disposal,Household residual waste,kg-CO2e/t,21.3,,BEIS,2021,GB,2021/11/18,Emission intensity for household residual waste (open-loop). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,open-loop
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1319]: From line 1319 - 1341, BEIS category is "Refuse". Suggest these go into "General waste" category.

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:1342]: From line 1342 - 1354 BEIS category is "Electrical items" - suggest category "Electrical Waste"

File: OpenEmissionFactorsDB.csv [hunk 5]
```diff
@@ -1277,3 +1277,142 @@ Energy,Fuel,fuel_type_coking_coal-fuel_use_na,Coking coal,kg-CO2e/t,3165.24,,BEI
...
+Waste,Construction Waste,waste_type_batteries-waste_use_disposal,Batteries,kg-CO2e/t,8.9,,BEIS,2021,GB,2021/11/18,Emission intensity for batteries (landfill). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,landfill
+Waste,Construction Waste,waste_type_metal_aluminium_cans_and foil_excluding_forming-waste_use_disposal,Metal: aluminium cans and foil (excl. forming),kg-CO2e/t,21.3,,BEIS,2021,GB,2021/11/18,Emission intensity for metal: aluminium cans and foil excluding forming (open-loop). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,open-loop
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1355]: From 1355 - 1370 BEIS category is "Metal" - suggest we use "Metal Waste"

File: OpenEmissionFactorsDB.csv [hunk 6]
```diff
@@ -1277,3 +1277,142 @@ Energy,Fuel,fuel_type_coking_coal-fuel_use_na,Coking coal,kg-CO2e/t,3165.24,,BEI
...
+Waste,Construction Waste,waste_type_scrap_metal_steel_cans-waste_use_disposal,Metal: steel cans,kg-CO2e/t,8.9,,BEIS,2021,GB,2021/11/18,Emission intensity for metal: steel cans (landfill). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,landfill
+Waste,Construction Waste,waste_type_average_plastics-waste_use_disposal,Average plastics,kg-CO2e/t,21.3,,BEIS,2021,GB,2021/11/18,Emission intensity for average plastics (open-loop). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,open-loop
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1371]: From line 1371 - 1406 BEIS category is "Plastic" suggest we us "Plastic Waste"

File: OpenEmissionFactorsDB.csv [hunk 7]
```diff
@@ -1277,3 +1277,142 @@ Energy,Fuel,fuel_type_coking_coal-fuel_use_na,Coking coal,kg-CO2e/t,3165.24,,BEI
...
+Waste,Construction Waste,waste_type_pvc_including_forming-waste_use_disposal,PVC (incl. forming),kg-CO2e/t,8.9,,BEIS,2021,GB,2021/11/18,Emission intensity for PVC including forming (landfill). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,landfill
+Waste,Construction Waste,waste_type_board-waste_use_disposal,Board,kg-CO2e/t,21.3,,BEIS,2021,GB,2021/11/18,Emission intensity for board waste (closed-loop). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,closed-loop
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1407]: From lines 1407 - 1418 the BEIS category is "Paper" suggest we use "Paper and Cardboard Waste"

**Review by patstirling — CHANGES_REQUESTED** [review]
See comments below on categorisation and other aspects.

A more general update is detailed here:

ID structure will need to be updated to reflect the method of disposal (this is not LCA activity, but an aspect of the method of waste disposal). This also means there are a lot of identical lines in here (activity does not provide uniqueness).

Suggest, for example:

`waste_type_metal_mixed_cans-disposal_method_open_loop`
`waste_type_metal_mixed_cans-disposal_method_closed_loop`
`waste_type_metal_mixed_cans-disposal_method_landfill`
`waste_type_metal_mixed_cans-disposal_method_combustion`

Any acronyms or initialisms used in the `ID` field (such as "hdpe") will need to be added to the glossary in this PR, [here](https://github.com/climatiq/Open-Emission-Factors-DB/blob/greenercode-BEIS-construction-waste/ID_STRUCTURE_GUIDANCE.md).

Activity will need to be updated to be in line with BEIS guidance (linked to you separately in [this doc](https://docs.google.com/spreadsheets/d/1yAvWLuov_CVdwnDfrZZiCrkNZQI-Tr-j/edit#gid=216654049)).

**Review by maryarbab — CHANGES_REQUESTED** [review]
To add one note to Pat's general comments: let's avoid using "average" in EFs to be consistent with the latest release. 
Also let's check the very high values...

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:1342]: I think from the description for lines 1342-1352 it is not clear that these are emissions from waste:

suggest "Emission intensity for WEEE - fridges and freezers (open-loop) waste disposal"

File: OpenEmissionFactorsDB.csv (hunk 5 above)
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:1355]: The same comment with the descriptions for lines 1353-1371, I think in the description if should be stated somewhere that it is "Emission intensity of waste..."

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]
Great job Stefan!! I have one additional comment - some descriptions are missing info that these are Emissions from waste. Would be great to adjust them! Thank you!

**Comment by patstirling** [comment]
NB. the name and description of this PR should be changed to reflect that more than construction waste is being added.

File: OpenEmissionFactorsDB.csv [hunk 8]
```diff
@@ -1277,142 +1277,142 @@ Energy,Fuel,fuel_type_coking_coal-fuel_use_na,Coking coal,kg-CO2e/t,3165.24,,BEI
...
+Waste,General Waste,waste_type_commercial_and_industrial-disposal_method_landfill,Commercial and industrial waste,kg-CO2e/t,467,,BEIS,2021,GB,2021/11/18,Emission intensity for commercial and industrial waste. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,gate_to_grave
+Waste,Electrical Items Waste,waste_type_weee_fridges_and_freezers-disposal_method_open_loop,WEEE (fridges and freezers),kg-CO2e/t,21.3,,BEIS,2021,GB,2021/11/18,Emission intensity for waste from electrical and electronic equipment (WEEE) - fridges and freezers. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,cradle_to_gate
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1342]: Suggest "Electrical Waste" for this category rather than "Electricl Items Waste"

**Review by patstirling — CHANGES_REQUESTED** [review]
Please review all name and description fields to make sure that the disposal method is reflected. E.g. for `waste_type_metal_mixed_cans-disposal_method_open_loop` would expect to see name: "Metal waste disposal (mixed cans) - open loop" in the name and similar detail in the description (currently all mixed cans lines are identical)

File: OpenEmissionFactorsDB.csv [hunk 9]
```diff
@@ -1277,3 +1277,142 @@ Energy,Fuel,fuel_type_coking_coal-fuel_use_na,Coking coal,kg-CO2e/t,3165.24,,BEI
...
+Waste,Electrical Items Waste,waste_type_batteries-disposal_method_landfill,Batteries waste,kg-CO2e/t,8.9,,BEIS,2021,GB,2021/11/18,Emission intensity for batteries waste. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,gate_to_grave
+Waste,Metal Waste,waste_type_metal_aluminium_cans_and foil_excluding_forming-disposal_method_open_loop,Metal: aluminium cans and foil waste (excl. forming),kg-CO2e/t,21.3,,BEIS,2021,GB,2021/11/18,Emission intensity for metal (aluminium cans and foil) waste Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,cradle_to_gate
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:1355]: waste_type_metal_aluminium_cans_and foil_excluding_forming-disposal_method_open_loop -> waste_type_metal_aluminium_cans_and_foil_excluding_forming-disposal_method_open_loop (there are bunch of these)

**Review by maryarbab — CHANGES_REQUESTED** [review]

**Review by AnastasiaLobanova — DISMISSED** [review]

File: OpenEmissionFactorsDB.csv [hunk 10]
```diff
@@ -1277,3 +1277,142 @@ Energy,Fuel,fuel_type_coking_coal-fuel_use_na,Coking coal,kg-CO2e/t,3165.24,,BEI
...
+Waste,Construction Waste,waste_type_aggregates-disposal_method_landfill,Aggregates waste,kg-CO2e/t,1.2,,BEIS,2021,GB,2021/11/18,Emission intensity for aggregates waste. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,gate_to_grave
+Waste,Construction Waste,waste_type_construction-disposal_method_open-loop,Construction waste,kg-CO2e/t,1,,BEIS,2021,GB,2021/11/18,Emission intensity for construction waste. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,cradle_to_gate
```
[the remark this answers is not in the document]
Comment (greenercode) [reply OpenEmissionFactorsDB.csv]: To be addressed in LCA reviewing. _For landfill, the factors in the tables include collection, transportation and landfill emissions (‘gate to grave’). For combustion and recycling, the factors consider transport to an energy recovery or materials reclamation facility only. This is in line with GHG Protocol Guidelines, with subsequent emissions attributed to electricity generation or recycled material production respectively._

**Review by maryarbab — DISMISSED** [review]

File: OpenEmissionFactorsDB.csv [hunk 11]
```diff
@@ -1277,3 +1277,142 @@ Energy,Fuel,fuel_type_coking_coal-fuel_use_na,Coking coal,kg-CO2e/t,3165.24,,BEI
...
+Waste,Plastic Waste,waste_type_pvc_including_forming-disposal_method_landfill,PVC waste disposal (incl. forming) - landfill,kg-CO2e/t,8.9,,BEIS,2021,GB,2021/11/18,Emission intensity for PVC waste disposal (incl. forming) - landfill. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,gate_to_grave
+Waste,Paper and Cardboard Waste,waste_type_board-disposal_method_closed_loop,Board waste disposal - closed-loop,kg-CO2e/t,21.3,,BEIS,2021,GB,2021/11/18,Emission intensity for board waste disposal - closed-loop. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,cradle_to_cradle
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1407]: For the "Paper and Cardboard Waste" section all instances of "board" in the ID, name or description be replaced with "cardboard" - this avoids confusion with other kinds of board (e.g. plaster)

**Review by patstirling — CHANGES_REQUESTED** [review]
Very close now - just a couple of things to fix in the guidance and one clarification in the Paper and Cardboard EFs

File: ID_STRUCTURE_GUIDANCE.md [hunk 12]
```diff
@@ -88,11 +88,11 @@ This category is for estimating the CO2 equivalent emissions of shipping goods b
...
 `sea_transport-vessel_type [e.g. "bulk_carrier"]- route_type_[e.g. "intra_mediterranean"]-vessel_length_[e.g. "gt-8000-teu"]-tonnage_[e.g. "gt_200000dwt"]-fuel_source_[e.g. "na" or "diesel"]`
 
-**Waste Transport**
+**Waste Disposal**
 
-This category is for estimating the CO2 equivalent emissions for generated waste. 
+This category is for estimating the CO2 equivalent emissions produced by the disposal of waste.
 
-`waste_type [e.g. "aggregates"]- disposal_method_[e.g."open-loop"]`
+`waste_type[e.g."aggregates"]-disposal_method_[e.g."open-loop"]`
```
Comment (patstirling) [inline ID_STRUCTURE_GUIDANCE.md:95]: Just missing that hyphen between waste type and example:

`waste_type_[e.g."aggregates"]-disposal_method_[e.g."open-loop"]`

File: OpenEmissionFactorsDB.csv [hunk 13]
```diff
@@ -1404,14 +1404,14 @@ Waste,Plastic Waste,waste_type_pvc_including_forming-disposal_method_open_loop,P
...
+Waste,Paper and Cardboard Waste,waste_type_cardboard-disposal_method_landfill,Cardboard waste disposal - landfill,kg-CO2e/t,1041.8,,BEIS,2021,GB,2021/11/18,Emission intensity for cardboard waste disposal - landfill. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,gate_to_grave
+Waste,Paper and Cardboard Waste,waste_type_paper_and_cardboard-disposal_method_closed_loop,Paper and Cardboard waste disposal - closed-loop,kg-CO2e/t,21.3,,BEIS,2021,GB,2021/11/18,Emission intensity for paper and cardboard waste disposal - closed-loop. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,cradle_to_gate
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1411]: Decapitalise "Cardboard" in `name` 1411-1414

**Review by patstirling — CHANGES_REQUESTED** [review]

**Review by patstirling — APPROVED** [review]

**Pull request merged**